src/client_list.c

```diff
@@ -219,6 +220,16 @@ int deleteFromList(char* service_name)
 			}
 			
 			ParodusPrint("Deleting the node\n");
+                        if(nn_shutdown(curr_node->sock, curr_node->endpoint) < 0)
+                        {
+                           ParodusError ("nn_shutdown socket=%d endpt=%d, err=%d\n", 
+				curr_node->sock, curr_node->endpoint, errno);
+                        }
+			if (nn_close (curr_node->sock) < 0)
```

Comment:
Add this same nn_close log at all places, like line 79

Reply:
Will add log.
But we don't need nn_shutdown on line 79 because in this case we did not connect successfully.